.github/workflows/windows-msvc-cuda.yml

```diff
@@ -0,0 +1,64 @@
+name: Windows-MSVC-CUDA (compile-only)
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
```

Comment:
you can remove this one
```suggestion
```

Reply:
do we? I thought we still have the test there from the other actions

Reply:
We might not even have a next release on that branch, since it has been deprecated since end of 2024 and will not be maintained after 2025.